Titanium.Web.Proxy/Network/CertificateManager.cs

```diff
@@ -73,20 +100,23 @@ internal CertificateEngine Engine
 
         private X509Certificate2 rootCertificate;
 
+        public bool only_load_rootCert { get; set; }
```

Comment:
please follow the guidelines: https://github.com/justcoding121/Titanium-Web-Proxy/wiki